tutorials/distributed_data_classification/distributed_data_classification.ipynb

```diff
@@ -0,0 +1,376 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Distributed Data Classification with Quality and Domain Classifiers\n",
+    "\n",
+    "The notebook demonstrates the use of two classifiers for distributed data classification, including quality and domain classifiers. The quality classifier is used to classify the quality of the data, while the domain classifier is used to classify the domain of the data. These classifers help with annotation which helps data blending for foundation model training. \n",
+    "\n",
+    "The classifiers are accelerated using CrossFit,(https://github.com/rapidsai/crossfit), a library that leverages intellegent batching and RAPIDS to accelerate the offline inference on large datasets."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "env: PYTHONWARNINGS=ignore\n"
+     ]
+    }
+   ],
+   "source": [
+    "#### Silence Warnings (HuggingFace internal warnings)\n",
+    "\n",
+    "%env PYTHONWARNINGS=ignore\n",
+    "import warnings\n",
+    "warnings.filterwarnings(\"ignore\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dask_cuda import LocalCUDACluster\n",
+    "from dask.distributed import Client\n",
+    "from nemo_curator import DomainClassifier, QualityClassifier\n",
+    "from nemo_curator.datasets import DocumentDataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cluster = LocalCUDACluster(rmm_async=True, rmm_pool_size=\"1GB\")\n",
+    "client = Client(cluster)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Define the data file paths "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input_file_path=\"/input_data_dir/\"\n",
+    "output_file_path = \"output_data_dir/\"\n",
+    "domain_model_path = \"domain_model.pth\"\n",
+    "quality_model_path = \"quality_model.pth\""
```

Comment:
I see you opened https://github.com/NVIDIA/NeMo-Curator/issues/72, I think once we add support for that we should use real links, paths, etc. so that the user can run it without changing anything.

Should we open an issue for this?

Reply:
Lets add to the issue for that to track we also change this file. Thanks for the suggestion. 👍🏼